addons/hr_expense/tests/common.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
remove this diff

Reply:
![image](https://github.com/odoo/odoo/assets/77900800/2881fce9-1cec-4e37-9ba8-7386ff167644)
Me with the code